.golangci.yml

```diff
@@ -83,7 +83,7 @@ linters-settings:
     min-len: 3
     min-occurrences: 3
     ignore-tests: true # re-enable once goconst 1.7.0+ merged into golangci-lint
-    ignore-strings: '.html'
+    ignore-strings: '\.html|true'
```

Comment:
If it's for single function, it's better to use local ignore in that function.

Reply:
I'll do some cleanup. It required goconst 1.7.0+.